packages/vuetify/src/components/VRangeSlider/VRangeSlider.ts

```diff
@@ -146,7 +147,7 @@ export default VSlider.extend({
         ref: 'track'
       }, children)
     },
-    genChildren () {
```

Comment:
no need for strict types on methods

Reply:
Copied from VSlider.ts. Should I leave it there?

Reply:
no

Reply:
fixed